skimage/data/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""
+lazy_loader makes it easy to load subpackages and functions on demand.
+
+"""
```

Comment:
Hey @ana42742, there is no `lazy_loader` function in this submodule so it's not really a fitting description yet. You could try looking through the other functions in [`skimage.data`](https://scikit-image.org/docs/stable/api/skimage.data.html) to get feel how this module might be best described. 

I think this is you second item of the list in #6761? That's alright :) but we recommend that each intern pick only one item of the list (as written in that issue). ;)